### PR TITLE
Update gRPC & HTTP stats

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: stats/gRPC.md [hunk 1]
```diff
@@ -9,117 +9,124 @@ Units are encoded according to the case-sensitive abbreviations from the [Unifie
...
 * Latencies are measures in float64 milliseconds, denoted "ms"
 * Sizes are measured in bytes, denoted "By"
-* Dimensionless values have unit "1"
+* Counts have unit "1"
 
 Buckets for distributions in default views are as follows:
 
 * Size in bytes: 0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864, 268435456, 1073741824, 4294967296
 * Latency in ms: 0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000
 * Counts (no unit): 0, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536
```
Comment (bogdandrutu) [inline stats/gRPC.md]: Should we mention that these are for the [sent, received]_messages_per_rpc?

**Comment by bogdandrutu** [comment]
Please create an issue to update the http specs to be consistent on names (if applicable).

**Comment by ramonza** [comment]
Working on HTTP now

**Comment by ramonza** [comment]
@bogdandrutu update with HTTP, PTAL

File: stats/HTTP.md [hunk 2]
```diff
@@ -22,70 +22,70 @@ Buckets for distributions in default views are as follows:
...
-| opencensus.io/http/server/response_bytes     | response_bytes | distribution | server_method, server_path                |
-| opencensus.io/http/server/latency            | latency        | distribution | server_method, server_path                |
-| opencensus.io/http/server/finished_count     | latency        | count        | server_method, server_path, server_status |
+| View name                                | Measure suffix | Aggregation  | Tags suffix                               |
+|------------------------------------------|----------------|--------------|-------------------------------------------|
+| opencensus.io/http/server/received_bytes | received_bytes | distribution | server_method, server_path                |
+| opencensus.io/http/server/sent_bytes     | sent_bytes     | distribution | server_method, server_path                |
+| opencensus.io/http/server/server_latency | server_latency | distribution | server_method, server_path                |
+| opencensus.io/http/server/finished_count | server_latency | count        | server_method, server_path, server_status |
```
Comment (bogdandrutu) [inline stats/HTTP.md]: why here finished and on client completed? Is that intentional?

File: stats/gRPC.md (hunk 1 above)
Comment (g-easy) [reply stats/gRPC.md]: Yes please, I'd like it to be really clear what "qps" is as a unit. :)

Comment (ramonza) [reply stats/gRPC.md]: MHz?

Comment (g-easy) [reply stats/gRPC.md]: None of the units here have the "/sec" divisor. Count is count until my monitoring system calculates the rate over time?

Comment (bogdandrutu) [reply stats/gRPC.md]: Correct, we produce cumulative data not rates. it is easy for backends to compute rates out of cumulative data.

**Pull request merged**